packages/loaders/src/styled/StyledSkeleton.ts

```diff
@@ -48,14 +47,16 @@ const retrieveSkeletonBackgroundColor = ({
   theme,
   isLight
 }: IStyledSkeletonProps & ThemeProps<DefaultTheme>) => {
-  if (isLight) {
-    return css`
-      background-color: ${rgba(getColorV8('background', 600 /* default shade */, theme)!, 0.2)};
-    `;
-  }
+  const backgroundColor = getColor({
+    theme,
+    hue: 'neutralHue',
+    transparency: theme.opacity[200],
+    light: { shade: isLight ? 500 : 700 },
```

Comment:
Should we deprecate `props.isLight` is favor of a more relevant `props.isInversed`?

From my research, it's only used once internally. The plan would be to support both until until v10.

Reply:
I think that could make sense. Let's see how the PR plays out with design feedback. If we do the rename, let's just rip the bandage and make it happen in v9.